README.md

```diff
@@ -35,6 +35,8 @@ The primary way to use this repository is as a Go library, but as a convenience
 it also contains a CLI tool called `terraform-config-inspect`, installed
 automatically by the `go get` command above, that allows viewing module
 information in either a Markdown-like format or in JSON format.
+You need to [explicitly add `$GOPATH/bin` to your `PATH`](https://github.com/actions/setup-go/issues/27#issuecomment-549102955)
```

Comment:
We should no need to link to a 3rd party repo for something fundamental like adding the binary to the PATH. It might be useful to mention the bin directory, (more specifically `GOBIN`), but `github.com/actions/setup-go` is unrelated to `github.com/hashicorp/terraform-config-inspect`.